CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
   * Before this change, the `bytes32` type was used for the enclave public keys, just supporting encryptors with public keys of that length (like the default NaCl)
   * For the EC encryptor, the encoded public key length is 91
 - `--tx-pool-hashes-max-size` option removed (deprecated in 22.1.3).
+- Block creation now returns `BlockCreationResult` which contains the new `Block` and `TransactionSelectionResults`. 
```

Comment:
this should not be a braking change, since you are not changing any configuration option, or plugin interface, so it could go under **Additions and Improvements**

Reply:
Distraction mistake 🤦🏼‍♂️ . Fixed, thanks 👍